projects/kit/components/progress/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,23 @@
+import {ChangeDetectionStrategy, Component, HostBinding, Input} from '@angular/core';
+import {tuiDefaultProp} from '@taiga-ui/cdk';
+import {TuiSizeS} from '@taiga-ui/core';
+
+@Component({
+    selector: 'progress[tuiProgressBar]',
+    template: '',
+    styleUrls: ['./progress-bar.component.less'],
+    host: {
+        '[style.--tui-progress-color]': 'color',
+    },
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TuiProgressBarComponent {
+    @Input()
+    @tuiDefaultProp()
+    color = 'var(--tui-primary)';
```

Comment:
Let's default it to `currentColor` and add `color: var(--tui-primary)` to host, this way you would be able to just style it with CSS, unless you want some fancy coloring like gradients and segments.

Reply:
done